keyboards/ydkb/grape/matrix.c

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
why does this have a custom matrix?

Reply:
The matrix rows are driven by a 74HC138 demultiplexer. My attempt to use the SN74x138 driver like the Wasdat, Wasdat Code, and AL1 does resulted in some keys not working. The existing custom matrix from the TMK fork that ydkb.io uses is fully functional.

Reply:
well, at the very least, this needs to be converted to the lite version of custom matrix, as zvecr indicated, and gpio commands should be used, not avr specifc ones.

Reply:
I was able to fix the lite version of the custom matrix with the SN74x138 driver, and all keys work now :)